lib/Scan.js

```diff
@@ -26,7 +26,12 @@ function Scan (Model, filter, options) {
     this.buildState = filter;
     this.filters[filter] = {name: filter};
   } else if (typeof filter === 'object'){
-    this.parseFilterObject(filter);
+    if (options && options.useRawAwsFilter) {
```

Comment:
I think it would be easier for other if we just check for FilterExpression in the filter.

```js
if(type filter.FilterExpression === 'string')
```